mozilla-mobile/fenix/app/src/main/res/values/strings.xml

```diff
@@ -464,6 +464,10 @@
     <string name="history_delete_all">Delete history</string>
     <!-- Text for the dialog to confirm clearing all history -->
     <string name="history_delete_all_dialog">Are you sure you want to clear your history?</string>
+    <!-- Text for the snackbar to confirm that multiple browsing history items has been deleted -->
+    <string name="history_delete_multiple_items_snackbar">History Deleted</string>
+    <!-- Text for the snackbar to confirm that a single browsing history item has been deleted -->
+    <string name="history_delete_single_item_snackbar">Deleted %1$s</string>
```

Comment:
@BMEduard @ekager 
Can we add the comment, here and in `bookmark_delete_folder_snackbar`, to explain what the placeable `%1$s` is?